### PR TITLE
DLS-4128 Upgrading Libraries to Play26

### PR DESCRIPTION
**Pull request by rajanimohan22**
Upgrading Libraries to Play26

### CODE REVIEW

File: app/uk/gov/hmrc/helptosaveapi/http/HttpClient.scala [hunk 1]
```diff
@@ -31,7 +31,7 @@ object HttpClient {
 
   private val rawHttpReads = new RawHttpReads
 
-  implicit class HttpClientOps(val http: uk.gov.hmrc.play.bootstrap.http.HttpClient) extends AnyVal {
+  implicit class HttpClientOps(val http: HttpClient) extends AnyVal {
```
Comment (calinursaciuc) [inline app/uk/gov/hmrc/helptosaveapi/http/HttpClient.scala:34]: Does it differentiate it from the object we have in scope?

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -8,11 +8,14 @@ object AppDependencies {
 
   val compile = Seq(
     ws,
-    hmrc                %% "bootstrap-backend-play-26" % "3.0.0",
-    hmrc                %% "mongo-caching"             % "6.14.0-play-26",
-    hmrc                %% "simple-reactivemongo"      % "7.30.0-play-26",
+    hmrc                %% "bootstrap-backend-play-26" % "5.2.0",
+    hmrc                %% "mongo-caching"             % "7.0.0-play-26",
+    hmrc                %% "simple-reactivemongo"      % "8.0.0-play-26",
+    hmrc                %% "http-verbs-play-26"        % "13.3.0",
```
Comment (calinursaciuc) [inline project/AppDependencies.scala:14]: remove the http verbs

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,16 +1,11 @@
...
+resolvers += Resolver.jcenterRepo
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.10.0")
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.1.0")
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.6.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.0.0")
```
Comment (calinursaciuc) [inline project/plugins.sbt:10]: this must be 2.1.0

Comment (calinursaciuc) [reply project/plugins.sbt:10]: boby must be 3.4.0

File: app/uk/gov/hmrc/helptosaveapi/http/HttpClient.scala (hunk 1 above)
Comment (rajanimohan22) [reply app/uk/gov/hmrc/helptosaveapi/http/HttpClient.scala:34]: No, its actually picking up the updated one only.

File: project/plugins.sbt (hunk 3 above)
Comment (rajanimohan22) [reply project/plugins.sbt]: Done

File: project/AppDependencies.scala [hunk 4]
```diff
@@ -8,11 +8,14 @@ object AppDependencies {
...
     "org.typelevel"     %% "cats-core"                 % "2.0.0",
-    "com.github.kxbmap" %% "configs"                   % "0.4.4"
+    "com.github.kxbmap" %% "configs"                   % "0.4.4",
+    compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.1" cross CrossVersion.full),
+    "com.github.ghik" % "silencer-lib" % "1.7.1" % Provided cross CrossVersion.full
```
[the remark this answers is not in the document]
Comment (rajanimohan22) [reply project/AppDependencies.scala:17]: service-integration-test is upgraded to the latest version but upgrading reactivemongo-test to 5.0.0-play-26 is failing. So upgraded just to 4.21.0-play-26.

**Review by calinursaciuc — APPROVED** [review]

**Pull request merged**